api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/EditN3GeneratorVTwo.java

```diff
@@ -78,8 +79,14 @@ public void subInMultiUris(Map<String,List<String>> varsToVals, List<String> n3t
         }         
     }
   
-    private boolean containsNullOrEmpty(List<String> values) {        
-        return values != null && ( values.contains(null) || values.contains("") );       
+    private boolean containsNullOrEmpty(List<String> values) {
+    	//return values != null && ( values.contains(null) || values.contains("") );
+    	if(values == null) return true ;
+    	if(values.isEmpty()) return true ;
```

Comment:
See my comment above ;-)  I was already thinking about it.  Sometimes I try to not over-collapse conditionals like that for improved readability and to make it slightly easier to comment out conditions during testing.  Functionally they are identical so there is no technical reason not to combine them.